test/e2e/logforwarding/fluent/fluent_test.go

```diff
@@ -15,18 +16,22 @@ import (
 const message = "My life is my message"
 
 var _ = Describe("[ClusterLogForwarder]", func() {
+	const basePort = 24224
 	var (
-		c *client.Test
-		f *Fixture
+		c           *client.Test
+		f           *Fixture
+		testOrdinal int
+		logTypes    = []string{"application", "infrastructure", "audit"}
```

Comment:
we can reuse this variable https://github.com/openshift/cluster-logging-operator/blob/master/pkg/apis/logging/v1/cluster_log_forwarder.go#L17